plan/physical_plan_builder.go

```diff
@@ -298,32 +328,18 @@ func (p *DataSource) convert2PhysicalPlan(prop *requiredProperty) (*physicalPlan
 			}
 		}
 	}
-	indices, includeTableScan := availableIndices(p.table)
-	if includeTableScan {
-		info, err = p.handleTableScan(prop)
-		if err != nil {
-			return nil, errors.Trace(err)
-		}
-	}
-	for _, index := range indices {
-		indexInfo, err := p.handleIndexScan(prop, index)
-		if err != nil {
-			return nil, errors.Trace(err)
-		}
-		if indexInfo.cost < info.cost {
-			info = indexInfo
-		}
-	}
-	return info, errors.Trace(p.storePlanInfo(prop, info))
+	return nil, nil
 }
 
-func addPlanToResponse(p PhysicalPlan, planInfo *physicalPlanInfo) *physicalPlanInfo {
-	np := p.Copy()
-	np.SetChildren(planInfo.p)
-	return &physicalPlanInfo{p: np, cost: planInfo.cost, count: planInfo.count}
+// addPlanToResponse creates a *physicalPlanInfo that adds p as parent of info.
+func addPlanToResponse(parent PhysicalPlan, info *physicalPlanInfo) *physicalPlanInfo {
+	np := parent.Copy()
+	np.SetChildren(info.p)
+	return &physicalPlanInfo{p: np, cost: info.cost, count: info.count}
 }
 
-// enforceProperty add an topN or sort or limit upon current operator.
+// enforceProperty creates a physical plan info satisfy the required property by adding
```

Comment:
physical plan info -> *physicalPlanInfo
We should be consistence with the comment of addPlanToResponse().